drizzlepac/outputimage.py

```diff
@@ -491,25 +498,33 @@ def writeFITS(self, template, sciarr, whtarr, ctxarr=None,
                             hdu.header.append(_card)
                 hdu.header['filename'] = self.outweight
                 hdu.header['CCDCHIP'] = '-999'
+
                 if self.wcs:
-                    pre_wcs_kw = self.find_kwupdate_location(hdu.header,'CD1_1')
+                    pre_wcs_kw = self.find_kwupdate_location(hdu.header, 'CD1_1')
                     # Update WCS Keywords based on PyDrizzle product's value
                     # since 'drizzle' itself doesn't update that keyword.
-                    addWCSKeywords(self.wcs,hdu.header, blot=self.blot,
+                    addWCSKeywords(self.wcs, hdu.header, blot=self.blot,
                                    single=self.single, after=pre_wcs_kw)
 
+                # explicitly remove EXTNAME, EXTVER from header - they are not allowed
+                for kw in RESERVED_KEYS[-2:]:
+                    if kw in hdu.header:
+                        del hdu.header[kw]
```

Comment:
Same reasoning as for `sciarr` above, this is not necessary: code on lines `#490-493` insures that `hdu` is not going to contain `'EXTNAME'` and `'EXTVER'`. Also, replace line `#493` with:
```python
hdu = fits.PrimaryHDU(data=whtarr, header=prihdu.header)
```

Reply:
Change was made in 4b565a7.